static/src/javascripts/projects/common/modules/commercial/contributions-service.js

```diff
@@ -360,7 +360,10 @@ export const renderBanner = (response) => {
             return fastdom.mutate(() => {
                 const container = document.createElement('div');
                 container.classList.add('site-message--banner');
-                container.classList.add('remote-banner', isPuzzlesBanner ? 'remote-banner--puzzles' : '');
+                container.classList.add('remote-banner');
+                if (isPuzzlesBanner) {
+                    container.classList.add('remote-banner--puzzles');
```

Comment:
```suggestion
                    container.classList.add('remote-banner', 'remote-banner--puzzles');
```

Reply:
The line above still adds `remote-banner`, so this should be fine as it is

Reply:
Indeed it does!